scripts/update-snarkyjs-bindings.sh

```diff
@@ -5,25 +5,27 @@ SNARKY_JS_PATH=src/lib/snarky_js_bindings/snarkyjs
 dune b src/lib/crypto/kimchi_bindings/js/node_js \
 && dune b src/lib/snarky_js_bindings/snarky_js_node.bc.js || exit 1
 
-mkdir -p "$SNARKY_JS_PATH"/dist/server/node_bindings
+mkdir -p "$SNARKY_JS_PATH"/dist/node/node_bindings
 cp _build/default/src/lib/crypto/kimchi_bindings/js/node_js/plonk_wasm* "$SNARKY_JS_PATH"/src/node_bindings/
-cp _build/default/src/lib/snarky_js_bindings/snarky_js_node*.js "$SNARKY_JS_PATH"/src/node_bindings/
+mv -f "$SNARKY_JS_PATH"/src/node_bindings/plonk_wasm.js "$SNARKY_JS_PATH"/src/node_bindings/plonk_wasm.cjs
+cp _build/default/src/lib/snarky_js_bindings/snarky_js_node.bc.js "$SNARKY_JS_PATH"/src/node_bindings/snarky_js_node.bc.cjs
+sed -i 's/plonk_wasm.js/plonk_wasm.cjs/' "$SNARKY_JS_PATH"/src/node_bindings/snarky_js_node.bc.cjs
 
 # better error messages
 # TODO: find a less hacky way to make adjustments to jsoo compiler output
 # `s` is the jsoo representation of the error message string, and `s.c` is the actual JS string
-sed -i 's/function failwith(s){throw \[0,Failure,s\]/function failwith(s){throw joo_global_object.Error(s.c)/' "$SNARKY_JS_PATH"/src/node_bindings/snarky_js_node.bc.js
-sed -i 's/function invalid_arg(s){throw \[0,Invalid_argument,s\]/function invalid_arg(s){throw joo_global_object.Error(s.c)/' "$SNARKY_JS_PATH"/src/node_bindings/snarky_js_node.bc.js
-sed -i 's/return \[0,Exn,t\]/return joo_global_object.Error(t.c)/' "$SNARKY_JS_PATH"/src/node_bindings/snarky_js_node.bc.js
-sed -i 's/function raise(t){throw caml_call1(to_exn$0,t)}/function raise(t){throw Error(t?.[1]?.c ?? "some error")}/' "$SNARKY_JS_PATH"/src/node_bindings/snarky_js_node.bc.js
+sed -i 's/function failwith(s){throw \[0,Failure,s\]/function failwith(s){throw joo_global_object.Error(s.c)/' "$SNARKY_JS_PATH"/src/node_bindings/snarky_js_node.bc.cjs
+sed -i 's/function invalid_arg(s){throw \[0,Invalid_argument,s\]/function invalid_arg(s){throw joo_global_object.Error(s.c)/' "$SNARKY_JS_PATH"/src/node_bindings/snarky_js_node.bc.cjs
+sed -i 's/return \[0,Exn,t\]/return joo_global_object.Error(t.c)/' "$SNARKY_JS_PATH"/src/node_bindings/snarky_js_node.bc.cjs
+sed -i 's/function raise(t){throw caml_call1(to_exn$0,t)}/function raise(t){throw Error(t?.[1]?.c ?? "some error")}/' "$SNARKY_JS_PATH"/src/node_bindings/snarky_js_node.bc.cjs
 
 # optimize wasm / minify JS (we don't do this with jsoo to not break the error message fix above)
-pushd "$SNARKY_JS_PATH"/src/node_bindings
-  wasm-opt --detect-features --enable-mutable-globals -O4 plonk_wasm_bg.wasm -o plonk_wasm_bg.wasm.opt
-  mv plonk_wasm_bg.wasm.opt plonk_wasm_bg.wasm
-  npx esbuild --minify --log-level=error snarky_js_node.bc.js > snarky_js_node.bc.min.js
-  mv snarky_js_node.bc.min.js snarky_js_node.bc.js
-popd
+# pushd "$SNARKY_JS_PATH"/src/node_bindings
```

Comment:
Why leave this in if we're just commenting it out?

Reply:
hm I guess you're right. I wanted to see whether not minifying the build makes the DX better -- there's trade-offs, so I'm not quite sure about it